src/manipulation/formatting/getTextFromFormattingEdits.ts

```diff
@@ -1,14 +1,7 @@
 ﻿import { SourceFile, TextChange } from "../../compiler";
 
 export function getTextFromFormattingEdits(sourceFile: SourceFile, formattingEdits: ReadonlyArray<TextChange>) {
-    // reverse the order
-    formattingEdits = [...formattingEdits].sort((a, b) => {
-        const aStart = a.getSpan().getStart();
-        const bStart = b.getSpan().getStart();
-        const difference = bStart - aStart;
-
-        return difference === 0 ? 1 : difference; // reverse when equal
-    });
+    formattingEdits = [...formattingEdits].reverse();
```

Comment:
the problem here was that the array was not reversed in case two items have the same start, this was breaking "move to a new file" refactor. Check fileTestChangesTest

Reply:
Hmmm... this no longer sorts though. Could you change it back to sort, but still fix the issue?

Reply:
done